config/component_macros.cmake

```diff
@@ -638,199 +633,6 @@ macro( register_parallel_test targetname numPE command cmd_args )
   unset( lverbose )
 endmacro()
 
-#----------------------------------------------------------------------#
```

Comment:
This seems unrelated to the timestep retirement -- I just want to be sure that it's an intentional change and not a diff from somewhere else!

Reply:
This was intentional, but now I'm having 2nd thoughts about it.  Marking this PR as WIP.  There is no urgency with this one.  

As a reminder, I will probably start the next release cycle on Monday.

Reply:
Okay - I fixed this PR so that it only has _timestep_ specific deletions.